app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt

```diff
@@ -95,6 +110,22 @@ class StoryActivityTest {
     Intents.release()
   }
 
+  @Test
+  fun testStoryActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
+      createStoryActivityIntent(
+        internalProfileId = internalProfileId,
+        topicId = TEST_TOPIC_ID_0,
+        storyId = TEST_TOPIC_ID_1
```

Comment:
The value shouild be `TEST_STORY_ID_1 `

Reply:
@justdvnsh Always reply to comments and mention if I have to review PR or not.

Reply:
oh okay.
sure.
PTAL